log/cloudwatch/output.go

```diff
@@ -60,27 +60,27 @@ type Administration interface {
 }
```

Comment:
this is for initializing a logger on startup, it does not really make sense to have this use anything except the base global logger. Also, these are largely debug messages since the log.Error is taking place right before we return the error.